src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -91,7 +95,7 @@ private Window AddTopLevelWindow()
                 Y = 0,
                 // By using Dim.Fill(), it will automatically resize without manual intervention
                 Width = Dim.Fill(),
-                Height = Dim.Fill()
+                Height = Dim.Fill(1)
```

Comment:
What does the `1` do here?

Reply:
`Fill(1)` means "leave one row margin on right".